metricflow-semantics/metricflow_semantics/test_helpers/config_helpers.py

```diff
@@ -23,3 +24,23 @@ class MetricFlowTestConfiguration(SnapshotConfiguration):
     # is created and persisted between runs. The source schema name includes a hash of the tables that should be in
     # the schema, so
     use_persistent_source_schema: bool
+
+
+class DirectoryPathAnchor:
```

Comment:
Writing a wrapper class around a builtin one-liner is cutting against one of Python's core design principles. Just make the constant use pathlib directly. `pathlib.Path(__file__).parent.directory`

Reply:
* Actually, I was looking to get rid of `__file__` to make it easier to use. See the added commit that does this. Could be made into a function though.
* I wanted to use these anchors for each of the semantic manifests. In getting the new package test configuration setup, not being able to auto-complete and relying on strings was error-prone. This is the n-th iteration that we've had to adjust this.